src/Playwright/Core/WritableStream.cs

```diff
@@ -53,42 +53,51 @@ internal WritableStream(IChannelOwner parent, string guid) : base(parent, guid)
     public Task CloseAsync() => Channel.CloseAsync();
 }
 
-internal class WritableStreamImpl : System.IO.Stream
+internal class WritableStreamWrapper : System.IO.Stream
 {
     private readonly WritableStream _stream;
 
-    internal WritableStreamImpl(WritableStream stream)
+    internal WritableStreamWrapper(WritableStream stream)
     {
         _stream = stream;
     }
 
-    public override bool CanRead => throw new NotImplementedException();
+    public override bool CanRead => false;
 
-    public override bool CanSeek => throw new NotImplementedException();
+    public override bool CanSeek => false;
 
     public override bool CanWrite => true;
 
-    public override long Length => throw new NotImplementedException();
+    public override long Length => throw new NotSupportedException();
 
-    public override long Position { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+    public override long Position { get => throw new NotSupportedException(); set => throw new NotSupportedException(); }
 
-    public override void Flush() => throw new NotImplementedException();
+    public override void Flush()
+    {
+    }
 
-    public override int Read(byte[] buffer, int offset, int count) => throw new NotImplementedException();
+    public override int Read(byte[] buffer, int offset, int count) => throw new NotSupportedException();
 
     public override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken) =>
-        throw new NotImplementedException();
+        throw new NotSupportedException();
 
-    public override void Close() => _stream.CloseAsync().ConfigureAwait(false);
+    protected override void Dispose(bool disposing)
+    {
+        if (disposing)
+        {
+            _stream.CloseAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+        }
 
-    public override long Seek(long offset, SeekOrigin origin) => throw new NotImplementedException();
+        base.Dispose(disposing);
+    }
 
-    public override void SetLength(long value) => throw new NotImplementedException();
+    public override long Seek(long offset, SeekOrigin origin) => throw new NotSupportedException();
```

Comment:
are all these C# changes caused by bumping Roslynator or how to they relate to the version bumps?

Reply:
These `Stream` based changes are strictly doc-based - no Roslynator detection there, maybe some async but I don't remember. I have posted link to `Stream` docs in commit description.

Reply:
Extracted to separate PR